app/models/concerns/user_access_key_overrides.rb

```diff
@@ -17,7 +17,24 @@ def valid_password?(password)
   def password=(new_password)
     @password = new_password
     return if @password.blank?
-    self.encrypted_password_digest = Encryption::PasswordVerifier.digest(@password).to_s
+    self.encrypted_password_digest = Encryption::PasswordVerifier.digest(@password)
+  end
+
+  def valid_personal_key?(normalized_personal_key)
+    Encryption::PasswordVerifier.verify(
+      password: normalized_personal_key,
+      digest: encrypted_recovery_code_digest
+    )
+  end
+
+  def personal_key
+    @personal_key
+  end
```

Comment:
This is in the revert commit which is code that was already merged.

But, I didn't do that because we did a touch of refactoring here to make the `personal_key` and `personal_key=` methods work like the `password` and `password=` methods since that is how we actually use it.